rosdep/python.yaml

```
@@ -7488,6 +7488,16 @@ python3-multimethod-pip:
   ubuntu:
     pip:
       packages: [multimethod]
+python3-munkres:
+  debian:
+    bullseye: [python3-munkres]
+    buster: [python3-munkres]
+  fedora: [python3-munkres]
+  opensuse: [python3-munkres]
+  ubuntu:
```

There is a suggestion in the [Files changed](https://github.com/ros/rosdistro/pull/34963/files) tab which indicates that this package is also part of RHEL.

```suggestion
  rhel:
    '*': [python3-munkres]
    '7': null
  ubuntu:
```